GettyImages.Api/WebHelper.cs

```diff
@@ -88,7 +76,7 @@ internal WebHelper(Credentials credentials, string baseAddress, DelegatingHandle
                         _credentials.ResetAccessToken();
                         using (var retryClient = new HttpClient(await GetHandlersAsync(headerParameters)))
                         {
-                            httpResponse = await retryClient.PostAsync(uri, new FormUrlEncodedContent(formParameters));
+                            httpResponse = await retryClient.PostAsyncWithRetryPolicy(uri, formContent);
```

Comment:
I'm re-using `formContent` here instead of `new`-ing up a `FormUrlEncodedContent` again.  Any issues there?